test/ci-performance-test/speed-test-config.json

```diff
@@ -58,6 +58,9 @@
       },
       "transitReluctance": {
         "RAIL": 1.0
+      },
+      "unprefer": {
```

Comment:
Not sure if we want to keep this but maybe @hannesj or @t2gran have an opinion.

Reply:
Oh, this is left-over. Speed test configurations for varying scenarios could actually live outside the cloned git repository path.